### ISSUE TITLE
Contextify crashes on Node 0.11

### ISSUE THREAD

**Issue by ColemanGariety** [sarajs/Sara#85]

**Comment by topokel** [comment]
I'm getting this on bcrypt etc too, a lot of native module devs just don't update their modules until the release is stable

**Comment by ColemanGariety** [comment]
@mkgh yeah I've noticed that. DO you know when 0.12 is expected to land? I might wait until then to release this framework. It's really close to done.

**Comment by topokel** [comment]
Judging from [this](https://github.com/joyent/node/issues/milestones) it'll be either in the coming weeks or a month or two, but that's just a guess. I couldn't find anything on the Joyent blog about an estimated date, just "0.12 is coming very soon"
